Dockerfile

```diff
@@ -8,7 +8,16 @@ COPY ./secrets /
 
 WORKDIR /src/
 
-CMD ["node", "/src/index.js"]
-
 COPY ./src/ /src/
-RUN npm install --production
+RUN npm install
+RUN npm rebuild
+
+FROM node:12.14.0-alpine 
+
+WORKDIR /src/
+
+COPY --from=builder /src/ .
+COPY ./secrets /
```

Comment:
Probably better to mount that directory at runtime. I don't think it's a good idea to copy anything named `secrets` or `keys` into a final docker image.
If this directory is to be used only for testing, then it's better to move it to `tests` dir.

Reply:
Yeah, I'm not 100% sure about what this is doing here and why it can't be mounted at runtime... @bushjames might have some better answers for us, but I suspect it simply makes life easier in non-production environments.

Reply:
@kirgene @lewisdaly the secrets directory gets copied to provide some "out of the box defaults". These are test secrets only, available on github etc... in production DFSPs must mount over this directory with their own secrets. These defaults facilitate testing without having to go through the pain of generating them.